packages/generator-one-app-module/generators/app/templates/base-child-module/package.json

```diff
@@ -6,34 +6,31 @@
   "scripts": {
     "prebuild": "npm run clean",
     "build": "bundle-module",
+    "watch:build": "npm run build -- --watch",
     "clean": "rimraf build",
     "prepare": "npm run build",
-    "lint": "eslint --ignore-path .gitignore --ext js,jsx .",
-    "watch:build": "npm run build -- --watch",
-    "test": "jest"
+    "test:lint": "eslint --ignore-path .gitignore --ext js,jsx,snap .",
+    "test:unit": "jest",
+    "test": "npm run test:lint && npm run test:unit"
   },
   "dependencies": {
-    "@americanexpress/one-app-bundler": "^6.0.0",
     "@americanexpress/one-app-router": "^1.0.0",
-    "content-security-policy-builder": "^2.1.0",
     "holocron": "^1.0.0",
-    "ip": "^1.1.5",
     "react": "^16.12.0"
   },
   "devDependencies": {
+    "@americanexpress/one-app-bundler": "^6.0.0",
+    "amex-jest-preset-react": "^5.0.2",
```

Comment:
This and all the other instances of this dependency should be version `^6.0.0`. Depends on https://github.com/americanexpress/amex-jest-preset-react/pull/7 to be merged and released though.

Reply:
6.0.0 is published to npm, however it isn't available in our internal registry. For now, will update with the understanding that it'll work error-free by tomorrow.

Reply:
for now, pointing to npm registry